src/pages/NavbarPage.tsx

```diff
@@ -14,49 +14,13 @@ const NavbarPage: FC = () => {
           </Navbar.Brand>
           <Navbar.Toggle />
           <Navbar.Collapse>
-            <ul className="mt-4 flex flex-col md:mt-0 md:flex-row md:space-x-8 md:text-sm md:font-medium">
-              <li>
-                <a
-                  href="#/navbars"
-                  className="block rounded bg-blue-700 py-2 pr-4 pl-3 text-white dark:text-white md:bg-transparent md:p-0 md:text-blue-700"
-                  aria-current="page"
-                >
-                  Home
-                </a>
-              </li>
-              <li>
-                <a
-                  href="#/navbars"
-                  className="block border-b border-gray-100 py-2 pr-4 pl-3 text-gray-700 hover:bg-gray-50 dark:border-gray-700 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-white md:border-0 md:p-0 md:hover:bg-transparent md:hover:text-blue-700 md:dark:hover:bg-transparent md:dark:hover:text-white"
-                >
-                  About
-                </a>
-              </li>
-              <li>
-                <a
-                  href="#/navbars"
-                  className="block border-b border-gray-100 py-2 pr-4 pl-3 text-gray-700 hover:bg-gray-50 dark:border-gray-700 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-white md:border-0 md:p-0 md:hover:bg-transparent md:hover:text-blue-700 md:dark:hover:bg-transparent md:dark:hover:text-white"
-                >
-                  Services
-                </a>
-              </li>
-              <li>
-                <a
-                  href="#/navbars"
-                  className="block border-b border-gray-100 py-2 pr-4 pl-3 text-gray-700 hover:bg-gray-50 dark:border-gray-700 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-white md:border-0 md:p-0 md:hover:bg-transparent md:hover:text-blue-700 md:dark:hover:bg-transparent md:dark:hover:text-white"
-                >
-                  Pricing
-                </a>
-              </li>
-              <li>
-                <a
-                  href="#/navbars"
-                  className="block py-2 pr-4 pl-3 text-gray-700 hover:bg-gray-50 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-white md:border-0 md:p-0 md:hover:bg-transparent md:hover:text-blue-700 md:dark:hover:bg-transparent md:dark:hover:text-white"
-                >
-                  Contact
-                </a>
-              </li>
-            </ul>
+            <Navbar.Link href="#/navbars" active>
+              Home
+            </Navbar.Link>
+            <Navbar.Link href="#/navbars">About</Navbar.Link>
+            <Navbar.Link href="#/navbars">Services</Navbar.Link>
+            <Navbar.Link href="#/navbars">Pricing</Navbar.Link>
+            <Navbar.Link href="#/navbars">Contact</Navbar.Link>
```

Comment:
I plan to add a disabled attr as well unless you think its not needed. Just let me know.

Reply:
I don't see it as a requirement but it is a nice to have I think

Reply:
okay sounds good. I added it with [this commit.](https://github.com/themesberg/flowbite-react/pull/16/commits/c0f3d188a40f0579cb15a5e22b351d64c806674c)